modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/BaseProductResourceImpl.java

```diff
@@ -411,7 +411,7 @@ public void delete(
 		throws Exception {
 
 		for (Product product : products) {
-			deleteProduct(product.getId());
+			deleteProduct(product.getProductId());
```

Comment:
Hi @matijapetanjek that is not allowed, you should never change base classes, I think you can override that method or you can ask for support on t-headless channel on Slack